custom_components/waste_collection_schedule/waste_collection_schedule/source/cc-montesquieu_fr.py

```diff
@@ -0,0 +1,180 @@
+# import datetime
+from datetime import date
+from waste_collection_schedule import Collection
+from dateutil.rrule import FR, MO, SA, SU, TH, TU, WE, rrule, weekday, WEEKLY
+import requests
+from bs4 import BeautifulSoup
+from pprint import pprint
+import dateparser
```

Comment:
dateparser is not a dependency of WCS so you cannot use it, Try to sanitise it and tranlslate the dates and use dateuil.parser.parse to parse the dates